### ISSUE TITLE
Incorrect display with browser Firefox 132.0.2 (64 Bit)

### ISSUE THREAD

**Issue by RaBla45** [newroco/emlviewer#65]
Thank you! I was really waiting for EMLviewer fpr NC28.
But ... In my Firefox 132.0.2 (64 Bit) on Win 11 and Win 10 there is unfortunately an incorrect display of the mail content.

Please, look here:
![Wrong display in firefox](https://github.com/user-attachments/assets/6afba7f1-cc96-40dc-bfa7-67edda704218)
The area of the mail header (To, Cc, Vo, Subject). It is mixed with the NextCloud window.
It looks fine in MS Edge. In Chrome as well.

A Firefox problem or a small change in the EML Viewer?

Best regards
Ralf

**Comment by hobbesjaap** [comment]
Same issue for me with Firefox on Linux. All good in Safari on macOS. I can view an email, but then I have to refresh the page/go to something else, in order to resume browsing.

**Comment by putt1ck** [comment]
@hobbesjaap can you confirm what version of FF you are running?

**Comment by hobbesjaap** [comment]
@putt1ck Of course, apologies. I'm running Mozilla Firefox Flatpak, 138.0.4 (64-bit) on Bazzite Linux.

Email viewer expands to full size and is rendered on top of the Nextcloud menu, but mouse clicks are only captured for the Nextcloud menu, so I can't close the email viewer after opening an .eml file, as the X is right on top of my profile icon.

**Comment by RaBla45** [comment]
> [@hobbesjaap](https://github.com/hobbesjaap) can you confirm what version of FF you are running?

I'm usung Win11 and firefox 139.0.1 (64 Bit).
There io no change in misfunction in EML-Viewer Version 2.0.1
I think nobody have time to solve the problem so i will close tzhe issue
